cilium/cmd/identity_get.go

```diff
@@ -40,7 +41,16 @@ func printIdentities(identities []*models.Identity) {
 
 		fmt.Fprintf(w, "ID\tLABELS\n")
 		for _, identity := range identities {
-			fmt.Fprintf(w, "%d\t%s\n", identity.ID, identity.Labels)
+			lbls := labels.NewLabelsFromModel(identity.Labels)
+			first := true
+			for _, lbl := range lbls.GetPrintableModel() {
+				if first {
+					fmt.Fprintf(w, "%d\t%s\n", identity.ID, lbl)
```

Comment:
A tabprinter might be more consistent with other CLI commands? (since you're changing anyway)

Reply:
I don't understand, this is already using a `tabwriter`?